django_prose_editor/static/django_prose_editor/default.js

```diff
@@ -110,7 +110,7 @@ function createEditor(textarea, config = null) {
       Heading.configure({ levels: config.headingLevels || [1, 2, 3, 4, 5] }),
     isTypeEnabled("horizontalRule", "horizontal_rule") && HorizontalRule,
     isTypeEnabled("italic", "em") && Italic,
-    isTypeEnabled("link") && Link,
+    isTypeEnabled("link") && Link.configure({ hideOpenInNewWindow: config.hideOpenInNewWindow || false }),
```

Comment:
I'm not completely sold on this. Is this a feature we have to expose here? I prefer steering people towards implementing their own presets if they want more configurability of the extensions since we don't have a way of configuring individual extensions except for the `headingLevels` (which predates the whole concept of extensions and presets)